test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/token/HapiTokenMint.java

```diff
@@ -68,7 +68,7 @@ protected HapiTokenMint self() {
 	@Override
 	protected long feeFor(HapiApiSpec spec, Transaction txn, int numPayerKeys) throws Throwable {
 		return spec.fees().forActivityBasedOp(
-				HederaFunctionality.TokenMint, this::usageEstimate, txn, numPayerKeys);
+				HederaFunctionality.TokenMint, SubType.TOKEN_FUNGIBLE_COMMON, this::usageEstimate, txn, numPayerKeys);
```

Comment:
Maybe we could leave it with the default implementation until we have E2E tests for this? (using the old `forActivityBasedOp`, skipping the `Subtype` argument)